crates/evm/fuzz/src/strategies/mutators.rs

```diff
@@ -504,30 +502,32 @@ mod tests {
     #[test]
     fn test_mutate_address() {
         let mut runner = TestRunner::new(Config::default());
+        let value = Address::random();
```

Comment:
can we do the other tests in this style as well? with the closure and one loop instead of every mutation in a separate test